src/main/java/no/unit/nva/model/EntityDescription.java

```diff
@@ -1,14 +1,18 @@
 package no.unit.nva.model;
 
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+import java.net.URI;
 import java.util.List;
 import java.util.Map;
 
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
 public class EntityDescription {
 
-    private String type;
+    private String publicationType;
```

Comment:
I suspect publication type is not a string, there are a limited set of possibilities, we should speak with PO, for now JournalArticle is the only type.

Reply:
Yes, I tried using the enum but I couldn't guarantee all possible mappings when Terje was testing various DOIs

Reply:
This got me to thinking about datacite and crossref, we should talk with JEG